backend/controller/console.go

```diff
@@ -40,6 +40,26 @@ func (*ConsoleService) Ping(context.Context, *connect.Request[ftlv1.PingRequest]
 	return connect.NewResponse(&ftlv1.PingResponse{}), nil
 }
 
+func verbSchemaString(sch *schema.Schema, verb *schema.Verb) string {
+	verbString := ""
+	_ = schema.Visit(verb, func(n schema.Node, next func() error) error {
+		switch n := n.(type) {
+		case *schema.Ref:
```

Comment:
How is this resolving transitive types? It looks like it only resolves a single ref, rather than recursively?

Reply:
ah yeah, lemme add a console test to verify/fix that.